client/src/fabric/tensegrity-fabric.ts

```diff
@@ -243,7 +235,7 @@ export class TensegrityFabric {
                             break
                     }
                 } else {
-                    this.physics.applyLocalFeatures(this.instance)
+                    this.roleFeatures.forEach(applyFeatureToInstance(this.instance))
```

Comment:
Is this gonna be a convention in the project now? It's interesting

Reply:
Things are getting clearer.